Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -62,6 +68,14 @@ ABenchmarkGymGameModeBase::ABenchmarkGymGameModeBase()
 	, bActorCountFailureState(false)
 	, bExpectedActorCountsInitialised(false)
 	, ActivePlayers(0)
+	, bHasActorMigrationCheckFailed(false)
+	, PreviousTickMigration(0)
+	, UXAuthActorCount(0)
+	, MigrationOfCurrentWorker(0)
+	, MigrationFrameCount(0)
+	, MigrationWindowFrameCount(9000) // About 5Mins(FPS=30 on worker)
```

Comment:
Why are we using ticks and not seconds/milli-seconds?
Character movement is time based, and when server frame rate struggles (we allow framerate to drop to 20 fps before failing), the number of ticks will be fewer.

Reply:
If an Actor moves like this over a short period of time, A->B->A, it's easy to miss statistics if use Seconds or Milliseconds.
The comment 5Mins only means about, not an exact time interval.